config/initializers/rateit_app.rb

```diff
@@ -1,2 +1,3 @@
 # frozen_string_literal: true
 # Rateitapp.plugin_manager.add('name_of_your_plugin')
+Rateitapp.oauth_domain = 'accounts.publicradio.org'
```

Comment:
Yes, I have it in the internal Rails app and remember placing it here too for testing - then forgot to solve that problem. How can I call on the internal app environment config files when testing the engine? Would adding an initializer to the spec dummy work?